crates/editor/src/editor.rs

```diff
@@ -8351,7 +8351,7 @@ impl Editor {
         }
     }
 
-    pub fn copy_permalink_to_line(&mut self, _: &CopyPermalinkToLine, cx: &mut ViewContext<Self>) {
+    fn get_permalink_to_line(&mut self, cx: &mut ViewContext<Self>) -> anyhow::Result<url::Url> {
```

Comment:
We shouldn't need to qualify this, as the `Result` in scope is already `anyhow`'s:

```suggestion
    fn get_permalink_to_line(&mut self, cx: &mut ViewContext<Self>) -> Result<url::Url> {
```

Reply:
I would not event slightly advocate for it in this repo (as it's far too gone to the other side of the spectrum), but nonetheless want to note: I generally really like when only `std::result::Result` gets unqualified as something coming from the prelude; the rest of the results seem to be much more readable when qualified.